fiks-meldingsformidler/src/main/java/no/difi/meldingsutveksling/ks/mapping/ForsendelseMapper.java

```diff
@@ -113,10 +112,8 @@ private NoarkMetadataFraAvleverendeSakssystem metaDataFrom(MeldingType meldingTy
         metadata.withJournaldato(journalDatoFrom(meldingType.getJournpost().getJpJdato()));
         metadata.withDokumentetsDato(journalDatoFrom(meldingType.getJournpost().getJpDokdato()));
         metadata.withTittel(meldingType.getJournpost().getJpOffinnhold());
-
         Optional<AvsmotType> avsender = getAvsender(meldingType);
         avsender.map(a -> a.getAmNavn()).ifPresent(metadata::withSaksbehandler);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1612)